src/index.ts

```diff
@@ -31,6 +31,9 @@ const wrappableRuntimeList = [
   "java11",
   "java17",
   "java21",
+  "dotnet6",
+  "dotnet7",
```

Comment:
Is `dotnet7` supported?

Reply:
It depends on your definition of "supported":
1. .NET 7 is no longer supported by Microsoft as of 5/14/24, so ideally customers shouldn't be using it.  However...
2. There's no technical reason why our Lambda instrumentation won't work for a .NET 7 Lambda function.

So, we think it's best to be as inclusive as possible here.